src/makeRobotAccountant.js

```diff
@@ -6,7 +6,27 @@
  */
 
 function makeRobotAccountant() {
-  // write code here
+  let callCount = 0;
+  let result = [];
+
+  return function getSum(num) {
+    callCount++;
```

Comment:
I wonder whether we should increment`callCount` in this place. Right now, the count will be incremented even if we don't invoke the last returned function.

I would put it in the last function, but it is not specified in the task description so I leave the decision to you. I just wanted to make sure that you are aware of this closure behavior.